eng/tools/sdk-suppressions/src/sdkSuppressions.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * This file contains types for the contents of the SDK suppressions file, sdk-suppressions.yml.
+ * For details, see: 
+ * - https://microsoftapc-my.sharepoint.com/:w:/g/personal/raychen_microsoft_com/EbOAA9SkhQhGlgxtf7mc0kUB-25bFue0EFbXKXS3TFLTQA
+ */
+
+import Ajv from 'ajv';
+import { SdkName, sdkLabels } from "./sdk.js";
+
+export const sdkSuppressionsFileName = "sdk-suppressions.yaml";
+
+export type SdkSuppressionsYml = {
+  suppressions: SdkSuppressionsSection;
+};
+
+export type SdkSuppressionsSection = {
+  [sdkName in SdkName]?: SdkPackageSuppressionsEntry[];
+};
+
+export type SdkPackageSuppressionsEntry = {
+  package: string;
+  "breaking-changes": string[];
+};
+
+export function validateSdkSuppressionsFile(suppressionContent: string | object | undefined | null): {
+  result: boolean,
+  message: string
+} {
+  if (suppressionContent === null) {
+    return {
+      result: false,
+      message: 'This suppression file is a empty file'
+    };
+  }
+
+  if (!suppressionContent) {
+    return {
+      result: false,
+      message: 'This suppression file is not a valid yaml.'
```

Comment:
Could you add the link of 'https://aka.ms/azsdk/sdk-suppression' to this error message?